Dockerfile

```diff
@@ -22,7 +22,7 @@ WORKDIR /obs/src/api
 # Installing the gem directly spares us from having to rpm package two different thor versions.
 RUN sudo gem.ruby2.5 install thor:0.19 foreman
 # Ensure there is a foreman command without ruby suffix
-RUN sudo ln -s /usr/bin/foreman.ruby2.5 /usr/bin/foreman
+#RUN sudo ln -s /usr/bin/foreman.ruby2.5 /usr/bin/foreman
```

Comment:
Was this change intended? If it was, please add a short explanation to the commit message. Otherwise just drop it:-)